packages/salesforcedx-vscode-core/src/conflict/conflictView.ts

```diff
@@ -98,17 +99,20 @@ export class ConflictView {
 
     this._treeView.onDidChangeVisibility(async () => {
       if (this.treeView.visible) {
-        this.updateEnablementMessage();
+        await this.updateEnablementMessage();
         await this.dataProvider.onViewChange();
       }
     });
 
     extensionContext.subscriptions.push(this._treeView);
   }
 
-  private updateEnablementMessage() {
+  private async updateEnablementMessage() {
+    const orgType = await getWorkspaceOrgType();
     this.treeView.message =
-      sfdxCoreSettings.getConflictDetectionEnabled() || this.diffsOnly
+      sfdxCoreSettings.getConflictDetectionEnabled() ||
```

Comment:
I find this hard to grok at a glance - so we're saying if conflict detection is enabled OR diffsOnly is true, OR the org is source tracked, then we shouldn't show an error about conflict detection not being enabled?

What's the scenario where we would care about the orgType but be able to get here not setting diffsOnly (diff the folder command) or having conflict detection enabled?

Reply:
Which I suppose is my way of asking if this is tested/where 😄